robocop/checkers/misc.py

```diff
@@ -258,3 +260,25 @@ def visit_LibraryImport(self, node):  # noqa
         if not node.name:
             return
         self.libraries.append(node)
+
+
+class ResourceFileChecker(VisitorChecker):
+    """ Checker for resource files. """
+    rules = {
+        "0912": (
+            "can-be-resource-file",
+            "No tests in '%s' file, consider renaming to '%s.resource'",
+            RuleSeverity.INFO
+        )
+    }
+
+    def visit_File(self, node):  # noqa
+        source = node.source if node.source else self.source
+        if source:
+            extension = Path(source).suffix
+            if '.robot' in extension and not any([isinstance(section, TestCaseSection) for section in node.sections]):
+                self.report("can-be-resource-file",
+                            Path(source).name,
+                            Path(source).stem,
+                            node=node)
+        super().visit_File(node)
```

Comment:
Do we need to visit it further?

Reply:
No, we don't. I accidentally copy-pasted.